pkgs/development/ocaml-modules/tsdl/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchgit, ocaml, findlib, ctypes, result, SDL2, pkgconfig, opam }:
+
+let
+  inherit (stdenv.lib) getVersion;
+
+  pname = "tsdl";
+  version = "0.9.0";
+
+in
+
+stdenv.mkDerivation {
+  name = "ocaml-${pname}-${version}";
+
+  src = fetchgit {
+    url = "http://erratique.ch/repos/tsdl.git";
+    rev = "refs/tags/v${version}";
+    sha256 = "0px6mksc9d8jlkjh7kj1g1jsak21vam2wpdna6xs8d6ihii8frnf";
+  };
+
+  buildInputs = [ ocaml findlib ctypes result pkgconfig opam ];
+  propagatedBuildInputs = [ SDL2 ];
+
```

Comment:
@vbgl Why should `ctypes` be propagated?
As far as I can tell, it's `libffi` that should be propagated from the `ctypes` derivation, not `ctypes` itself from its dependents.

I have tested using `tsdl` from an OCaml package that doesn't use `ctypes`, and I haven't experienced any dependency problems.